### PR TITLE
Lausanne multiscale Atlases

### PR DESCRIPTION
**Pull request by frheault**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # FreeSurfer pipeline
 ===================
 
-Run the FreeSurfer recon-all pipeline.
+Run the FreeSurfer recon-all pipeline and create the customize freesurfer,
+brainnetome and glasser connectivity atlas in native space.
```
Comment (mdesco) [inline README.md:5]: It is also possible to run this nextflow if your FreeSurfer is already computed.

File: USAGE [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
 
-Run recon-all on all encountered subjects
+Run recon-all on all subjects and create the customize freesurfer,
+brainnetome and glasser connectivity atlas in native space.
+
+The input can be the input for freesurfer, in this case the pipeline will start
+with recon-all. The input can also be the output folders from freesurfer, in
+this case the pipeline will start with the creation of the customize atlas.
+Both options can be used together, but running freesurfer on CBRAIN is recommanded
+to reduce usage on ComputeCanada's clusters.
```
Comment (mdesco) [inline USAGE]: and it is faster and better optimized for large number of subjects.

File: USAGE [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+
+This pipeline must be used with a custom freesurfer singularity, several
+hard-to-get files are needed as well as custom scripts. The singularity
+container is available on braindata.
```
Comment (mdesco) [inline USAGE:18]: Can you add somewhere that output will be both in MNI space and native space?

**Review by mdesco — APPROVED** [review]
After minor edits

**Comment by frheault** [comment]
The pipeline is not doing that

**Pull request closed without merging**